components/Users/Memberships/index.js

```diff
@@ -120,7 +122,68 @@ const getState = (membership) => {
   return 'aktiv'
 }
 
+
+const SET_AUTO_PAY = gql`
+  mutation setMembershipAutoPay($id: ID!, $autoPay: Boolean!) {
+    setMembershipAutoPay(id: $id, autoPay: $autoPay) {
+      id
+      autoPay
+    }
+  }
+`
+
+function AutoPayToggle (membership) {
+
+  if (!membership.autoPayIsMutable) return (
+    <React.Fragment>
+      <DT>
+        Automatisch abbuchen
+      </DT>
+      <DD>
+        { membership.autoPay ? 'YES' : 'NO' }
+      </DD>
+    </React.Fragment>
+  )
+
+  return (
+    <Mutation
+      mutation={SET_AUTO_PAY}
+      variables={{ id: membership.id, autoPay: !membership.autoPay }}
+    >
+      {(mutation, {loading, error}) => {
+
+        return (
+          <React.Fragment>
+            <DT>
+              Automatisch abbuchen
+              {error && (
+                <React.Fragment>
+                  <br/>
+                  <span style={{color: colors.error}}>
+                    {error.message}
+                  </span>
+                </React.Fragment>
+              )}
+            </DT>
+            <DD style={{position: 'relative'}}>
+              { loading && (<Spinner/>) }
+              <Dropdown
+                black
+                label=''
+                items={[{value: true, text: 'Yes'}, {value: false, text: 'No'}]}
+                value={membership.autoPay}
+                onChange={mutation}
```

Comment:
For compatibility with the native dropdown `value` (translated to `option.value`) needs to be a string. But you can add arbitrary data to an item and access it in the onChange handler:
```suggestion
                items={[{value:'1', datum: true, text: 'Yes'}, {value:'0', datum: false, text: 'No'}]}
                value={membership.autoPay}
                onChange={(item) => mutation({ variables: { id: membership.id, autoPay: item.datum }})}
```

Reply:
Note to self: 

```
value={membership.autoPay ? '1' : '0'}
```

Reply:
@tpreusse wouldn't it make sense to move this logic into the high-level `<Dropdown />` component?

Reply:
Let's open a ticket but work with it as is now. An alternativ would be to use `item.text` as value of the select option (needs to be uniq too for users to differentiate).